DQMOffline/Configuration/python/DQMOffline_cff.py

```diff
@@ -158,7 +158,7 @@
 
 
 DQMOfflinePrePOGExpress = cms.Sequence( DQMOfflineTracking *
-                                 DQMOfflineMUO *
```

Comment:
https://github.com/cms-sw/cmssw/blob/6da11027b7ec7b8220f266521178030faccc3d71/DQMOffline/Configuration/python/DQMOffline_cff.py#L132

and

https://github.com/cms-sw/cmssw/blob/6da11027b7ec7b8220f266521178030faccc3d71/DQMOffline/Muon/python/muonMonitors_cff.py#L25-L30

I think this removed more than it was meant to (see https://cms-talk.web.cern.ch/t/missing-folder-in-streamexpress-run2024i-express-v2-dqmio/52718/1). @fabiocos @ptcox

Reply:
Indeed, I missed this, will fix it ASAP. Apologies.